README.md

```diff
@@ -14,6 +14,15 @@ tweetnacl.h -> 02aee579efbe4f12ad8ffb23c6772841
 ## Fixes
 Change integer size in commit [b98e0ca](https://github.com/RIOT-OS/tweetnacl/commit/b98e0ca8beb9b802b5c0492d88846bd9de9946eb)
 
+Fix a warning on line 58 about comparison between two different integer
+sizes.
```

Comment:
So, if while compiling for a 8bit platform on one side you have `int` and in the inner function you have a u32 variable there would not be an error, but we would be using a variable bigger than necessary (yes, its only 2 bytes, but still...).

If your argument is based on the warning output given by the compiler (which is about signedness), then I can put the complete output from the compiler.